code/modules/clothing/neck/syndicate_pet_collar.dm

```diff
@@ -0,0 +1,85 @@
+/obj/item/clothing/neck/petcollar/syndie	//For true gamer felinids
+	name = "suspicious pet collar"
+	desc = "A suspicious looking pet collar designed for inferior species."
+	icon_state = "petcollar_syndie"
+	flash_protect = 3
+	bang_protect = 5
+	var/pouncing = FALSE
+	var/pounce_nextuse = 0
+	var/pounce_cooldown = 50
+
+/obj/item/clothing/neck/petcollar/syndie/equipped(mob/user, slot)
+	pouncing = FALSE
+	var/mob/living/carbon/human/H = user
+	if(!istype(H))	//Give the spell to non-catpeople but make it do something worse
+		return
+	if(slot == SLOT_NECK)
+		H.AddSpell(new /obj/effect/proc_holder/spell/pounce)
+
+/obj/item/clothing/neck/petcollar/syndie/dropped(mob/user)
+	. = ..()
+	var/mob/living/carbon/human/H = user
+	H.RemoveSpell(/obj/effect/proc_holder/spell/pounce)
+
+//================================
+//========= Pounce Spell =========
+//================================
```

Comment:
the area for the spell

Reply:
no i just mean why the hell did you add this

Reply:
I just wanted to separate it off from the neck item

Reply:
Oohhhhhh

Reply:
Because it's funny

Reply:
i get overcoming the stun thing, though i am still opposed. you could include the "pounce" shit on an admeme-only version of the collar, but it's fetish content and useful, and race specifics should not be very useful

Reply:
It's not fetish content, but I get the thing with the stuns being somewhat useful, although it is a pretty terrible item since it does damage to you when you hit someone and if you are holding anything, you drop it too